xilem_web/web_examples/after_update/Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
-name = "after_update"
+name = "after_updtae"
```

Comment:
Typo, but maybe we should call this example something like "Raw DOM access", so that it's obvious what it's about.